### ISSUE TITLE
No confirmation message or validation when saving settings for Paypal Express

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2135]
### Reaction version
1.1.0

### Expected behavior
Should receive a "Settings Saved" toast if saved, or validation errors if fields are empty

### Actual Behavior
Nothing happens

### Steps to Reproduce the Behavior
1. Login as admin
1. Enter settings for Paypal Express
1. Observe that no message is shown
1. Remove values from fields (e.g. password)
1. Click save
1. Observe that no validation error is shown

**Comment by zenweasel** [comment]
Actually it looks like none of the payment settings forms validate against their schema.

**Comment by zenweasel** [comment]
Seems like this is being caused by this line in the `PackageConfig` schema.
```
  settings: {
    type: Object,
    optional: true,
    blackbox: true
  },
```
If I remove the `blackbox: true` line, everything starts validating again. I can also fix this by just removing the inheritance of `PackageConfig` for things like `BraintreePackageConfig`. However, some of the packages (e.g. Avalara) still validate correctly, but they are one more layer deep (there is a TaxPackageConfig` schema in the inheritance chain`)

**Comment by efalayi** [comment]
I was able to reproduce this issue. Other payment methods display a toast message when an update is made to the payment settings (e.g Braintree, Stripe). The issue here is the method that displays the toast is probably not called in Paypal function when an update is made.

**Comment by efalayi** [comment]
@zenweasel none of the payment method settings have validations.

**Comment by zenweasel** [comment]
@efalayi I'm aware. We will be converting all these to React shortly so we can revisit then.

**Comment by zenweasel** [comment]
Closed via #3012
